### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Sinclair-Speccy** [Yggdrasil-LGX/Yggdrasil-LGX#1]
I was just wondering but where did you get the source code from? I have a few CDs of it myself from a site and I was wondering how you are compiling it

**Comment by azraeljd** [comment]
I don't have the source code. If I had the source code, it would be too old to be compatible with current standards anyway, therefore I will be starting the project from scratch.

The source code for the last versions of Yggdrasil from 1993-1995 was found by a YouTuber named NCommander on ibiblio and he managed to fix some issues and got it working on a VM. I might add that to a separate repository at some point.

**Comment by Sinclair-Speccy** [comment]
I remember a site where I got my Yggdrasil CD-ROM images from used to have the source code as I was curious whenever you got it from there or somewhere else

**Comment by azraeljd** [comment]
Yes, it's probably from ibiblio (sunsite, I guess was the name back then). By the way, sorry for the horribly late reply.

Please stay tuned for the new release! I have successfully made a working system, only lacking a few VBox drivers for Xorg.

**Comment by Sinclair-Speccy** [comment]
Actually it wasn't ibiblio ^^;;

**Comment by tallship** [comment]
I've got a few unopened shrink-wrapped copies of Yggdrasil LGX Plug and Play Linux beginning with the 0.98 Linux kernel, as well as a few different source archives I've located over the years from various distribution points.

Is this also going to be distributed as a LiveCD, like the original alpha and beta m

Also worked on the *Arena browser* which is included in the distro - will we be including this in any upcoming general release?

Last question,

Has Adam Richter weighed in at all on this? **Yggdrasil Computing** owned the name, artwork (parts that weren't public domain like Icarus), etc, that went along with the retail packs and the two editions of the original **Linux Bible - The GNU Testament**.

I'm very interested in possibly contributing to this project in some capacity and had considered discussing just such a thing with Adam.

Do let me know, and the best immediate way to reach me either be via Matrix at `#tallship:matrix.org`.

Thanks, and this sounds exciting!

⛵

.

**Comment by azraeljd** [comment]
> I've got a few unopened shrink-wrapped copies of Yggdrasil LGX Plug and Play Linux beginning with the 0.98 Linux kernel, as well as a few different source archives I've located over the years from various distribution points.
> 
> Is this also going to be distributed as a LiveCD, like the original alpha and beta?

Yes, LiveCD, with automated GUI installer.

> Also worked on the _Arena browser_ which is included in the distro - will we be including this in any upcoming general release?

Due to Arena being a bit behind, being compatible with only HTML 4 and lower, I am considering not to for the time being, though I am interested in maintaining it further as well.

> Last question,
> 
> Has Adam Richter weighed in at all on this? **Yggdrasil Computing** owned the name, artwork (parts that weren't public domain like Icarus), etc, that went along with the retail packs and the two editions of the original **Linux Bible - The GNU Testament**.

No, I have not contacted Adam Richter. All artworks and non-PD/non-free (in terms of licensing) materials will be removed.

> I'm very interested in possibly contributing to this project in some capacity and had considered discussing just such a thing with Adam.

Thank you for your interest, contributions are very welcome.

**Comment by Sinclair-Speccy** [comment]
May I ask but how are you compiling it to have a newer kernel? I wanted to try myself for fun

**Comment by azraeljd** [comment]
> May I ask but how are you compiling it to have a newer kernel? I wanted to try myself for fun

I am continuing this project from scratch, with the same name, purpose, goals, and concepts as the original Yggdrasil LGX distribution. So, I am compiling everything from scratch. I am currently trying to make a Buildroot-based build system for this purpose. The last build successfully built a full system with Linux 6.0-rc5, Bash, Vim, and other tools, but lacking a working X.Org server and a desktop.
